docs/src/reference/asciidoc/appendix/release-notes/8.0.0-rc1.adoc

```diff
@@ -14,6 +14,28 @@ https://github.com/elastic/elasticsearch-hadoop/pull/1808[#1808]
 - Update Gradle wrapper to 7.3
 https://github.com/elastic/elasticsearch-hadoop/pull/1809[#1809]
 
+- Update log4j version
+https://github.com/elastic/elasticsearch-hadoop/pull/1828[#1828]
```

Comment:
@masseyke @jbaiera This isn't in the list at https://github.com/elastic/elasticsearch-hadoop/pull/1859#issuecomment-1011088314, but I think it should be included here.

Reply:
Sounds good to me. We don't actually ship log4j, so it's just a compile-time dependency. But probably worth mentioning even that since the log4j bug was such a high-profile thing.